packages/fast-components/src/badge/badge.ts

```diff
@@ -0,0 +1,45 @@
+import { attr, FastElement } from "@microsoft/fast-element";
+
+export enum BadgeSize {
+    small = "small",
+    medium = "medium",
+    large = "large",
+}
+
+export class Badge extends FastElement {
+    @attr({ attribute: "fill" })
+    public fill: string;
+
+    // Revisit this once we have a better
+    // story for ensuring proper contrast
+    // from author defined `fill`
+    @attr({ attribute: "color" })
+    public color: string;
+
+    @attr
+    public size: BadgeSize;
+    private sizeChanged(): void {
+        switch (this.size) {
```

Comment:
I think you will also need to remove the previous size class.

Reply:
Yeah, I think what we want to do here is just remove the current size before we enter the switch.

```
this.classList.remove(this.size);
```